packages/arb-token-bridge-ui/synpress.cctp.config.ts

```diff
@@ -0,0 +1,163 @@
+  const ethAmountSepolia = utils.parseEther('0.01')
+  const ethAmountArbSepolia = utils.parseEther('0.002')
+  const ethPromises: (() => Promise<void>)[] = []
+  const usdcPromises: (() => Promise<void>)[] = []
+
+  if (tests.length === 2) {
+    ethPromises.push(() =>
+      fundEth({
+        address: userWalletAddress,
+        provider: sepoliaProvider,
+        sourceWallet: localWallet,
+        amount: ethAmountSepolia
+      })
+    )
+    ethPromises.push(() =>
+      fundEth({
+        address: userWalletAddress,
+        provider: arbSepoliaProvider,
+        sourceWallet: localWallet,
+        amount: ethAmountArbSepolia
+      })
+    )
+    usdcPromises.push(
+      () =>
+        fundUsdc({
+          address: userWalletAddress,
+          provider: sepoliaProvider,
+          networkType: 'parentChain',
+          sourceWallet: localWallet,
+          amount: usdcAmount
+        }),
+      () =>
+        fundUsdc({
+          address: userWalletAddress,
+          provider: arbSepoliaProvider,
+          networkType: 'childChain',
+          sourceWallet: localWallet,
+          amount: usdcAmount
+        })
+    )
+  } else if (tests[0].includes('deposit')) {
+    ethPromises.push(() =>
+      fundEth({
+        address: userWalletAddress,
+        provider: sepoliaProvider,
+        sourceWallet: localWallet,
+        amount: ethAmountSepolia
+      })
+    )
+    usdcPromises.push(() =>
+      fundUsdc({
+        address: userWalletAddress,
+        provider: sepoliaProvider,
+        networkType: 'parentChain',
+        sourceWallet: localWallet,
+        amount: usdcAmount
+      })
+    )
+  } else {
+    ethPromises.push(() =>
+      fundEth({
+        address: userWalletAddress,
+        provider: arbSepoliaProvider,
+        sourceWallet: localWallet,
+        amount: ethAmountArbSepolia
+      })
+    )
+    usdcPromises.push(() =>
+      fundUsdc({
+        address: userWalletAddress,
+        provider: arbSepoliaProvider,
+        networkType: 'childChain',
+        sourceWallet: localWallet,
+        amount: usdcAmount
+      })
+    )
+  }
+
+  await Promise.all(ethPromises.map(fn => fn()))
+  await Promise.all(usdcPromises.map(fn => fn()))
+}
+
+export default defineConfig({
+  ...getCommonSynpressConfig(shouldRecordVideo),
+  e2e: {
+    async setupNodeEvents(on, config) {
+      logsPrinter(on)
+
+      await fundWallets()
+
+      config.env.PRIVATE_KEY = userWallet.privateKey
+      config.env.PRIVATE_KEY_CCTP = process.env.PRIVATE_KEY_CCTP
+      config.env.SEPOLIA_INFURA_RPC_URL = SEPOLIA_INFURA_RPC_URL
+      config.env.ARB_SEPOLIA_INFURA_RPC_URL = ARB_SEPOLIA_INFURA_RPC_URL
+      config.env.CUSTOM_DESTINATION_ADDRESS =
+        await getCustomDestinationAddress()
+
+      setupCypressTasks(on, { requiresNetworkSetup: false })
+      synpressPlugins(on, config)
+      return config
+    },
+    baseUrl: 'http://localhost:3000',
+    specPattern: tests,
+    supportFile: 'tests/support/index.ts'
```

Comment:
This can also be a part of commonSynpressConfig, and reused no?

Reply:
tests differ between the two. The only common part is the baseUrl and the supportFile, I'm not sure it's worth creating a common file for that?